File: scratch/add_option_e.py
import re

with open("c:/Workspace/petrobras-quest/src/components/aulas/portugues/data/crase-quizzes.ts", "r", encoding="utf-8") as f:
    content = f.read()

# Substituir ] após a opção D por opção E e ]
# Matches:
#      { label: "D", valor: "..." },
#    ],
new_content = re.sub(
    r'( *\{ *label: *"D".*?\}),?\n *],',
    r'\1,\n      { label: "E", valor: "Nenhuma das alternativas anteriores" },\n    ],',
    content
)

with open("c:/Workspace/petrobras-quest/src/components/aulas/portugues/data/crase-quizzes.ts", "w", encoding="utf-8") as f:
    f.write(new_content)

print("Done")

File: scratch/add_option_e.js
const fs = require('fs');
let content = fs.readFileSync('c:/Workspace/petrobras-quest/src/components/aulas/portugues/data/crase-quizzes.ts', 'utf8');

content = content.replace(/(\{\s*label:\s*"D"[^\}]+\}),?\s*\n\s*\],/g, '$1,\n      { label: "E", valor: "Nenhuma das alternativas anteriores" },\n    ],');

fs.writeFileSync('c:/Workspace/petrobras-quest/src/components/aulas/portugues/data/crase-quizzes.ts', content, 'utf8');
console.log('Replaced occurrences!');
